Decode/EventHeaderEnumerator.cs

```diff
@@ -1181,7 +1173,7 @@ public EventHeaderItemInfo GetItemInfo(ReadOnlySpan<byte> eventDataSpan)
         /// whether the event contains any trailing data (data not described by the
         /// decoding information). Up to 7 bytes of trailing data is normal (padding
         /// between events), but 8 or more bytes of trailing data might indicate some
-        /// decodingStyle of encoding problem or data corruption.
+        /// kind of encoding problem or data corruption.
```

Comment:
Note: Visual Studio "rename Symbol" was incorrectly done with the "update comments" option enabled, incorrectly updating a few comments. Fix by restoring the comment.